.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -3,11 +3,11 @@ name: Feature request
 about: JanusGraph can't do all the things, but maybe it can do your things.
 
 ---
-
-For discussion on internal implementation details of JanusGraph or proposing a [major new feature or subproject](https://docs.janusgraph.org/development/#development-decisions) use the [janusgraph-dev](https://groups.google.com/forum/#!forum/janusgraph-dev).
-
+<!--
+For discussion on internal implementation details of JanusGraph or proposing a [major new feature or subproject](https://docs.janusgraph.org/development/#development-decisions) use the [janusgraph-dev](https://lists.lfaidata.foundation/g/janusgraph-dev/topics).
```

Comment:
We could include https://github.com/JanusGraph/janusgraph/discussions/categories/ideas also